src/Model/Tag.php

```diff
@@ -325,6 +325,32 @@ private static function getTypeForHash(string $hash)
 		}
 	}
 
+	/**
+	 * Create implicit mentions for a given post
+	 *
+	 * @param integer $uri_id
+	 * @param integer $parent_uri_id
+	 */
+	public static function createImplicitMentions(int $uri_id, int $parent_uri_id)
+	{
+		if ($uri_id == $parent_uri_id) {
+			return;
+		}
+
+		// Always mention the direct parent author
+		$parent = Item::selectFirst(['author-link', 'author-name'], ['uri-id' => $parent_uri_id]);
+		self::store($uri_id, self::IMPLICIT_MENTION, $parent['author-name'], $parent['author-link']);
```

Comment:
Does this mean that the replied to author might not be notified if `explicit_mentions` are enabled and the mention in the body is removed?

Reply:
Yes. But this behaviour hasn't changed. It had been this way before as well. With that user feature the user has got the total control about who gets mentioned.

Reply:
Mentions and notifications are different. Since we are a thread-based software, I believe directly replying to someone should always notify them, whether they are mentioned in the body or not. It may look confusing for post-based software like Mastodon and Pleroma, but it still is the right thing to do.

Reply:
Well, like I said: I haven't changed the general behaviour, but only the backend. In fact I would like to do this completely different in the future (in a way that will combine all our different ways).